api/v1/swagger.yaml

```diff
@@ -1790,6 +1790,586 @@ paths:
+      tags: [ generators, writers ]
+      description: >-
+        Add one or more users to an generator’s list of authorized writers. If the Refocus configuration parameter `useAccessToken` is set to `true`, you must include an `Authorization` request header with your [JSON Web Token](https://tools.ietf.org/html/rfc7519) (JWT) as the value. You can get a token using `POST /v1/register` or `POST /v1/token`.
+      operationId: postGeneratorWriters
+      parameters:
+        -
+          name: key
+          in: path
+          description: >-
+            The id or name of the generator for which the authorized list of users
+            are to be obtained.
+          required: true
+          type: string
+        -
+          name: queryBody
+          description: Request body.
+          in: body
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+            description: >
+              User name or id
+      responses:
+        201:
+          description: >-
+            One or more of the valid users were added to the list of authorized writers.
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/GeneratorWriterResponse"
+              description: >-
+                The users having write permission to the generator
+        400:
+          $ref: "#/responses/400"
+        403:
+          $ref: "#/responses/403"
+        404:
+            $ref: "#/responses/404"
+        default:
+          $ref: "#/responses/genericError"
+
+  # ---------------------------------------------------------------------------
+
+  /generators/{key}/writers/{userNameOrId}:
+    get:
+      security:
+        - jwt: []
+      summary: Determine whether a user is an authorized writer for an generator
+      tags: [ generators, writers ]
+      description: >-
+        Determine whether a user is an authorized writer for an generator. If the Refocus configuration parameter `useAccessToken` is set to `true`, you must include an `Authorization` request header with your [JSON Web Token](https://tools.ietf.org/html/rfc7519) (JWT) as the value. You can get a token using `POST /v1/register` or `POST /v1/tokens`.
+      operationId: getGeneratorWriter
+      parameters:
+        -
+          name: key
+          in: path
+          description: >-
+            The id or name of the generator for which the authorized list of users
+            are to be obtained.
+          required: true
+          type: string
+        -
+          name: userNameOrId
+          in: path
+          description: >-
+            The username or the Id of the user
+          required: true
+          type: string
+      responses:
+        200:
+          description: >-
+            Success, returns the users having write permission to the generator.
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/UsersResponse"
```

Comment:
GET /generators/{key}/writers/{userNameOrId} should be a single item or 404. Updated the description and schema.